optuna/visualization/matplotlib/_timeline.py

```diff
@@ -62,10 +63,12 @@ def _get_timeline_plot(info: _TimelineInfo) -> "Axes":
     if len(info.bars) == 0:
         return ax
 
+    # According to the `ax.barh` docstring, using list[timedelta] as width and
+    # list[datetime] as left is supported, but mypy does not recognize it.
     ax.barh(
```

Comment:
[nit] I added the reference.
```suggestion
    # cf. https://github.com/matplotlib/matplotlib/blob/v3.10.1/lib/matplotlib/axes/_axes.py#L2701-L2836
    ax.barh(
```